packages/SidePanel/src/components/Content/Content.styles.js

```diff
@@ -0,0 +1,8 @@
+import styled from "styled-components";
+import stylers from "@paprika/stylers";
+
+export const contentStyles = styled.div`
+  margin-left: ${stylers.spacer(-3)};
```

Comment:
I just wonder if this is the best solution to apply negative margins on this component, rather than just setting the padding to 0 on the parent itself, and having a prop like `hasZeroPadding` to set it to zero padding? I know we want to avoid breaking api changes but it might be a good time to?

Reply:
good point, I wonder he is trying to not break the current implementation, but I think we should not use margins at all, use the parent container and set it to padding 0, and the the `<Content />` can have padding=8 16 whatever its need. 

How do you feel about breaking the current implementation @mikrotron @oscarkwan

Reply:
Break it. The version is `< 0`, that means it's not yet stable.  We shouldn't be too reluctant to make breaking _improvements_.

Reply:
this component is so WIP we had to give it a negative version